pkg/KubeArmorOperator/enforcer/enforcer.go

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package enforcer
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	probe "github.com/daemon1024/bpflsmprobe/probe"
+	"go.uber.org/zap"
+	"k8s.io/kubectl/pkg/util/slice"
+)
+
+// GetAvailableLsms Functio
+func GetAvailableLsms() []string {
+	return []string{"bpf", "selinux", "apparmor"}
+}
+
+// DetectEnforcer: detect the enforcer on the node
+func DetectEnforcer(lsmOrder []string, PathPrefix string, log zap.SugaredLogger) string {
+	supportedLsms := []string{}
+	lsm := []byte{}
+	lsmPath := PathPrefix + "/sys/kernel/security/lsm"
+
+	if _, err := os.Stat(filepath.Clean(lsmPath)); err == nil {
+		lsm, err = os.ReadFile(lsmPath)
+		if err != nil {
+			log.Info("Failed to read /sys/kernel/security/lsm " + err.Error())
+			goto probeBPFLSM
+		}
+	}
+
+	supportedLsms = strings.Split(string(lsm), ",")
+
+probeBPFLSM:
+	if !slice.ContainsString(supportedLsms, "bpf", nil) {
+		err := probe.CheckBPFLSMSupport()
+		if err == nil {
+			supportedLsms = append(supportedLsms, "bpf")
+		} else {
+			log.Warnf("BPF LSM not supported %s", err.Error())
+		}
+	}
```

Comment:
```suggestion
probeLSM:
	if !slice.ContainsString(supportedLsms, "bpf", nil) {
		err := probe.CheckBPFLSMSupport()
		if err == nil {
			supportedLsms = append(supportedLsms, "bpf")
		} else {
			log.Warnf("BPF LSM not supported %s", err.Error())
		}
	}
	
	if !slice.ContainsString(supportedLsms, "apparmor", nil) {
		// Read file `/sys/module/apparmor/parameters/enabled`
	}
```

This is based on https://kubernetes.io/docs/tutorials/security/apparmor/#before-you-begin, You might want to include the link in code too.

This will help us probe for AppArmor even if the security LSM file is not there.